<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVoiceKeyer</class>
 <widget class="QDialog" name="DialogVoiceKeyer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voice Keyer Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_voiceK4">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_voiceK3">
        <property name="text">
         <string>3</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_voiceK3</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_voiceK3"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_voiceK2">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_voiceK4"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton_voiceK1">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_voiceK2"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_voiceK1"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_voiceK4">
        <property name="text">
         <string>4</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_voiceK4</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButton_voiceK3">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_voiceK1">
        <property name="text">
         <string>1</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_voiceK1</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_voiceK2">
        <property name="text">
         <string>2</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_voiceK2</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_voiceK5">
        <property name="text">
         <string>5</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_voiceK5</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_voiceK5"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButton_voiceK5">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Audio Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QSlider" name="horizontalSlider_audioLevel">
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_audioLevel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>10</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="comboBox_audioDevice"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_voiceK1</tabstop>
  <tabstop>pushButton_voiceK1</tabstop>
  <tabstop>lineEdit_voiceK2</tabstop>
  <tabstop>pushButton_voiceK2</tabstop>
  <tabstop>lineEdit_voiceK3</tabstop>
  <tabstop>pushButton_voiceK3</tabstop>
  <tabstop>lineEdit_voiceK4</tabstop>
  <tabstop>pushButton_voiceK4</tabstop>
  <tabstop>lineEdit_voiceK5</tabstop>
  <tabstop>pushButton_voiceK5</tabstop>
  <tabstop>comboBox_audioDevice</tabstop>
  <tabstop>horizontalSlider_audioLevel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogVoiceKeyer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogVoiceKeyer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
